site/next.config.js

```diff
@@ -32,6 +32,11 @@ const nextConfig = {
         destination: "/contact/submitted",
         permanent: true,
       },
+      {
+        source: "/dashboard/create-schema",
+        destination: "/dashboard?create-schema=true",
+        permanent: true,
+      },
```

Comment:
Btw I think this is fine for now, but I suspect the preference for `/dashboard/create-schema` was not just to be a redirect, but to be the actual endpoint of the modal (so that the browser URL is more readable and doesn't contain query parameters). (Correct me if I'm wrong @nonparibus )

I think this could be accomplished by relocating `dashboard.page.tsx` to `/dashboard/[...slugs].page.tsx`, and opening the modal when `router.query.slugs === ["create-schema"]`.

But appreciate that's a lot more work than a simple redirect, so we can do this down the line.

Reply:
Updated to use the dynamic routes approach here => 6309ee3

Reply:
Nice, this could be further extended to clear the `/create-schema` part of the url when the modal is closed, and to add it when the "create a schema" button is clicked on the dashboard page, but this is fine for now.